databases/week2/MoreQueries.sql

```diff
@@ -1,40 +1,25 @@
-
+USE hyfData;
 -- Get all the tasks assigned to the user who's email ends in @spotify.com
 SELECT 
-    task.title, user.email
-FROM 
-    task
-JOIN 
-    user
-WHERE
-    user.email LIKE "%@spotify.com";
+task.title, user.email
+FROM task
+INNER JOIN user WHERE user.email LIKE "%@spotify.com";
```

Comment:
Good work re-structuring the code. However my main concern here is that you are not explicitly saying where the inner join should happen. This is something that should always be stated. for example: 

SELECT
task.title,
user.email,
FROM task
INNER JOIN user **ON task.user_id = user.id**
WHERE user.email LIKE "%@spotify.com";

When the join condition is not defined MySQL will choose the join key, which in this case works but its unlikely to work every time.